lib/backend/syncersv1/updateprocessors/bgpconfigprocessor.go

```diff
@@ -100,3 +105,39 @@ var svcClusterIpsToString = func(value interface{}) interface{} {
 
 	return strings.Join(ipCidrs, ",")
 }
+
+// return JSON encoded string of CommunityKVPair
```

Comment:
I'm not quite sure if I'm doing it the right way by marshalling the struct. Let me know whats the best option her, I'll revisit this again while doing confd changes, I'll have more understanding on it by then.